src/db/inout/importcsv.cc

```diff
@@ -1381,12 +1379,13 @@ void
         std::string msg{"column '" + m + "' is missing, import is aborted"};
         log_error("%s", msg.c_str());
         LOG_END;
-        bios_throw("request-param-bad", m.c_str(), std::string("<missing column'").append(m).append("'>").c_str(),
-            std::string("<column '").append(m).append("' is present in csv>").c_str());
+        std::string msg1 = zsys_sprintf (TRANSLATE_ME ("<missing column '%s'>", m.c_str ()));
```

Comment:
This is a memleak to me, as it passes char * to string copy constructor, which won't free it. It might be possible that compiler uses some smart moving or whatever, but I doubt that.